src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js

```diff
@@ -31,7 +43,29 @@ angular.module('catan')
                         var edgeId = angular.element(event.target).closest(DrawMapService.EDGE_SELECTOR).attr('edge-id');
                         SelectMapObjectService.select('edge', edgeId);
                     });
+
                 }
             };
 
+            //TODO: move that code to some helper?
+            function highlightHexes(element, map, dice, timeout) {
+                var hexesToHighlight = map.hexes.filter(function(hex) {
+                    return (dice === 7) ? hex.robbed : !hex.robbed && hex.dice === dice;
```

Comment:
dice - current gotten dice number here; hex - each hex on the map.  Why 'hex.robbed' will never return 'true'?

If dice != 7 we highlight hexes which have dice = gotten dice number (only if they are not robbed). If dice = 7, we highlight "robbers" (currently we highlight not exactly robbers but hexes which are robbed, but I think it doesn't matter for now). Isn't it right?